platform/posix/man/yaml2ck.1

```diff
@@ -0,0 +1,58 @@
+.TH "yaml2ck" 1 "12 Aug 2022" "yaml2ck" \" -*- nroff -*-
+.ad l
+.nh
+.SH NAME
+yaml2ck \- convert Cantera YAML input files to Chemkin-II-format mechanisms
```

Comment:
```suggestion
yaml2ck \- convert Cantera YAML input files to Chemkin-format mechanisms
```

`man` text should be equivalent to `yaml2ck --help`. I don't think that the `-II` suffix matters?